patch_roadmap.sh

```
@@ -0,0 +1,4 @@
+sed -i '/- GHCR is verified as a secondary runtime for the publication image/c\- GHCR is verified as a secondary runtime for the publication image (`ghcr.io/effortlessmetrics/tokmd`, `verified-public` for `v1.14.0` as of 2026-06-25).' docs/ROADMAP.md
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)